### DISCUSSION TITLE
How to handle blocked packages in solution

### DISCUSSION THREAD

**Discussion by verzada** [NuGet/Home#11371] [Q&A]
My team has been testing out blocking vulnerable nuget packages in our package repository.
What we've noticed is, that it is a taxing task to figure out which package needs to be upgraded, since nuget cannot resolve the dependencies of what is blocked. 

So I'm wondering what is the best approach for 
.NET projects (and its dependency tree)
and 
.Net Framework projects
?

**Comment by crusader-mike** [comment]
Can this help:
```
nuget restore -Verbosity detailed -DisableParallelProcessing 
```
?
